src/process/routes/index.js

```diff
@@ -0,0 +1,11 @@
+import Pages from '../../show/pages';
+import * as Paths from './paths';
+
+const Home = {
+  element: Pages.Home,
+  path: Paths.HOME_PATH
+};
+
+const routes = [Home];
+
+export default routes;
```

Comment:
I don't see you using this routes file. Either delete it, either use it properly.

Reply:
okay boso